tools/dev-scripts/data/barmen.json

```diff
@@ -0,0 +1,121 @@
+[
+    {
+        "id": 0,
```

Comment:
May be, it's better if you let mysql take care of the ids ?! I have some `Duplicate entry \'1\' for key \'PRIMARY\'` error during the script excecution ...

Reply:
It should be fixed know, I set higher ids (to not collide with Auto Increment) and fixed the godfather association